Makefile

```diff
@@ -180,3 +202,41 @@ $(EMBEDMD): $(LOCALBIN)
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+.PHONY: protogen
+protogen: protoc ## Generate protobuf files.
+	cd internal/sci ; protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative sci.proto
+
+.PHONY: protoc
+protoc: $(PROTOC) ## download and install protoc.
+$(PROTOC): $(LOCALBIN)
+	test -s $(LOCALBIN)/protoc || \
+		curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip -o /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip
+		unzip /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip -d /tmp/protoc/ && \
+		cp /tmp/protoc/bin/protoc $(LOCALBIN)/protoc && \
+		rm -rf /tmp/protoc/
+		rm /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip
+		go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}
+
+# curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-${PLATFORM}-${ARCH}.zip && \
+# unzip mv $(LOCALBIN)/protoc
+
+.PHONY: build_gcp_installer
+build_gcp_installer: ## Build the GCP installer.
+	docker build ./install -t substratus-installer
+
+.PHONY: gcp_installer_up
+gcp_installer_up: build_gcp_installer ## invoke the GCP installer to build all infra.
+	docker run -it \
+		-v ${HOME}/.kube:/root/.kube \
+		-e PROJECT=$(shell gcloud config get project) \
+		-e TOKEN=$(shell gcloud auth print-access-token) \
```

Comment:
for dev I generally add this:
```
-e INSTALL_OPERATOR=no
```
that prevents the operator from being installed such that I can run make dev locally

Reply:
Gotcha. I tidied this up so the long and repetitive docker command is captured in a variable and just the final bits are manipulated. `make gcp_installer_up_dev` is now your new friend!